scripts/exrap_dump.sh

```diff
@@ -1254,49 +1255,8 @@ set -x
 # determine local system name and type if available
 # -------------------------------------------------
 SITE=${SITE:-""}
-sys_tp=${sys_tp:-$(getsystem.pl -tp)}
-getsystp_err=$?
-if [ $getsystp_err -ne 0 ]; then
-   msg="***WARNING: error using getsystem.pl to determine system type and phase"
-   set +u
-   [ -n "$jlogfile" ] && $DATA/postmsg "$jlogfile" "$msg"
-   set -u
-fi
-echo sys_tp is set to: $sys_tp
-
-if [ "$sys_tp" = 'Cray-XC40' -o "$SITE" = SURGE -o "$SITE" = LUNA ]; then
-   launcher=${launcher:-"aprun_cfp"}
-else
-   launcher=${launcher:-"cfp"}  # iDataPlex
-fi
-if [ "$launcher" = aprun_cfp ]; then
-   #  Get compute node count:  Subtract one from the total number of unique
-   #  hosts to account for the MAMU node that runs serial portion of job
-   typeset -i nodesall=$(echo -e "${LSB_HOSTS// /\\n}"|sort -u|wc -w)
-   typeset -i ncnodes=$(($nodesall-1)) # we want compute nodes only
-   if [ $ncnodes -lt 1 ]; then
-      set +x
-      echo
-      echo " ######################################################## "
-      echo " --> Could not get positive compute node count for aprun! "
-      echo " --> Check that BSUB directives included a reservation    "
-      echo "                   request for one or more compute nodes. "
-      echo " --> @@ F A T A L   E R R O R @@   --  ABNORMAL EXIT      "
-      echo " ######################################################## "
-      echo
-      set -x
-      $DATA/err_exit "***FATAL: Check if compute nodes were allocated"
-   fi
-elif [[ "$launcher" = cfp && -z "$LSB_HOSTS" ]]; then
-   set +x
-   echo
-   echo "You requested the cfp poe launcher but are not running under LSF!!"
-   echo "You must run under LSF to use cfp option on IBM.  Exiting..."
-   echo
-   set -x
-   $DATA/err_exit
-fi
-if [ "$launcher" = cfp -o "$launcher" = aprun_cfp ]; then
+launcher=${launcher:-"cfp"}
+if [ "$launcher" = cfp ]; then #-o "$launcher" = aprun_cfp ]; then
```

Comment:
I'd nix everything to the right of the "then". Those are WCOSS1 Cray settings -- not relevant here, and just confusing to be there, even commented.
#-o "$launcher" = aprun_cfp ]; then